commands/utils/git.go

```diff
@@ -146,3 +166,16 @@ func (gm *GitManager) IsClean() (bool, error) {
 
 	return status.IsClean(), nil
 }
+
+func createBasicAuth(token string) *http.BasicAuth {
+	return &http.BasicAuth{
+		Username: "username", // The username can be anything except an empty string
```

Comment:
Let's put the comment above the line is descrives.

... except an empty string --> ... except for an empty string